Makefile

```diff
@@ -759,6 +759,15 @@ endif
 	LIBS += $(NCLIB)
 endif
 
+ifneq "$(SCOTCH)" ""
+	SCOTCH_FCINCLUDES += -I$(SCOTCH)/src/include
+	SCOTCH_LIBS += -L$(SCOTCH)/lib -lscotch -lscotcherr
```

Comment:
It may depend on the version on Scotch, but when I installed v7.0.7, the header files are in `.../include` (rather than in `.../src/include`, and the libraries are in `.../lib64` rather than in `.../lib`.